cmd/run.go

```diff
@@ -119,6 +121,10 @@ func runE(cmd *cobra.Command, _ []string) error {
 	if exclude != "" {
 		excludeRE = regexp.MustCompile(exclude)
 	}
+	var tagRE *regexp.Regexp
+	if tag != "" {
+		tagRE = regexp.MustCompile(tag)
```

Comment:
We shouldn't be using `MustCompile` on user input. Could you change those to handle errors properly and report to the user?